tests/MultipleWhereClauseTest.php

```diff
@@ -0,0 +1,54 @@
+<?php
+
+namespace BaoPham\DynamoDb\Tests;
+
+use BaoPham\DynamoDb\DynamoDbModel;
+use BaoPham\DynamoDb\DynamoDbQueryBuilder;
+
+/**
+ * Class MultipleWhereClauseTest
+ * @package BaoPham\DynamoDb\Tests
+ * @author Matthew Collins <matthew@instanceco.com>
+ */
+class MultipleWhereClauseTest extends DynamoDbTestCase
```

Comment:
I'd prefer to have this test in the existing test file to ensure it covers all types, composite or non-composite. I also don't think this needs a separate file.

Reply:
Sounds good. I moved the test case to both the composite and non-composite model test classes. Let me know if I've misinterpreted your request, or if there's anything else you'd like to see here. Thanks!